src/Arcus.WebApi.Logging/SkipRequestTrackingAttribute.cs

```diff
@@ -1,11 +0,0 @@
-﻿using System;
-
-namespace Arcus.WebApi.Logging
-{
-    /// <summary>
-    /// Represents an endpoint attribute that indicates which endpoints should be withhold from request tracking.
-    /// </summary>
-    public class SkipRequestTrackingAttribute : Attribute
```

Comment:
Careful here, we'll need to be backwards compatible on this one

Reply:
No need, I double checked it and we didn't released this attribute yet, so we're free to change it. 👍